TACC/Core-Portal
# task/WP-100: Display all jobAttributes via getJobDisplayInformation

## Overview

The view details in Job History currently shows input parameters that have isHidden: true properties. This code fixes that issue as well as adds few more jobAttributes to the Output section.

## Related

* [WP-100](https://jira.tacc.utexas.edu/browse/WP-100)

## Changes

* added logic to filter isHidden: true properties
* added few more jobAttributes to render


## Testing

1. Go to History > Jobs
2. Click on 'View Details' under Job Details and check to see if the hidden parameters are shown or not under Inputs
3. Verify if there are two more attributes in Outputs

## UI

![After filtering out hidden properties](https://github.com/TACC/Core-Portal/assets/54924215/74c4fa3a-a160-4624-922b-82528b9a7bac)

![image](https://github.com/TACC/Core-Portal/assets/54924215/114454e0-73f3-4938-af2f-fb5199013f7d)



## Notes
